### PR TITLE
ethereum support and economy model merge develop into master

### PR DESCRIPTION
**Pull request by vchain007**
merge

### CODE REVIEW

**Review by JerryGavin20 — APPROVED** [review]

File: chainstorage/src/main/java/org/vision/core/store/DynamicPropertiesStore.java [hunk 1]
```diff
@@ -273,13 +292,13 @@ private DynamicPropertiesStore(@Value("properties") String dbName) {
...
     } catch (IllegalArgumentException e) {
-      this.saveWitnessPayPerBlock(100000L);//0.1VS
+      this.saveWitnessPayPerBlock(144000L);//0.144000VS
     }
 
     try {
       this.getWitnessStandbyAllowance();
     } catch (IllegalArgumentException e) {
-      this.saveWitnessStandbyAllowance(7_200_000_000L);//1VS
+      this.saveWitnessStandbyAllowance(115_200_000_000L);//115200 VS
```
Comment (vchain007) [inline chainstorage/src/main/java/org/vision/core/store/DynamicPropertiesStore.java:301]: plz change @JerryGavin20 
getWitnessStandbyAllowance（for maintain） and getWitness123PayPerBlock are independent by proposal getChangeDelegation，

File: chainstorage/src/main/java/org/vision/core/store/DynamicPropertiesStore.java [hunk 2]
```diff
@@ -732,6 +811,83 @@ private DynamicPropertiesStore(@Value("properties") String dbName) {
...
+    }
+
+    try {
+      this.getSpreadMintLevel();
+    } catch (IllegalArgumentException e) {
+      this.saveSpreadMintLevel(3);
+    }
+
+    try {
+      this.getAllowSpreadMintLevelProp();
```
Comment (vchain007) [inline chainstorage/src/main/java/org/vision/core/store/DynamicPropertiesStore.java:881]: getAllowSpreadMintLevelProp never use plz confirm
@darylccc  @theway001 @BeginningRhyme-vision

File: chainstorage/src/main/java/org/vision/core/store/DynamicPropertiesStore.java [hunk 3]
```diff
@@ -2162,6 +2453,190 @@ public long getAllowBlackHoleOptimization() {
...
+
+  public long getSpreadMintPayPerBlock() {
+    return Optional.ofNullable(getUnchecked(SPREAD_MINT_PAY_PER_BLOCK))
+            .map(BytesCapsule::getData)
+            .map(ByteArray::toLong)
+            .orElse(256000L);
+  }
+
+  public boolean supportSpreadMint() {
+    return true;
```
Comment (vchain007) [inline chainstorage/src/main/java/org/vision/core/store/DynamicPropertiesStore.java:2483]: @JerryGavin20 @BeginningRhyme-vision @theway001 
no proposal and storage？

**Review by vchain007 — COMMENTED** [review]
too many need change

**Review by JerryGavin20 — APPROVED** [review]

**Review by darylccc — APPROVED** [review]

**Review by newjava1 — APPROVED** [review]

**Pull request merged**